eslint.config.js

```diff
@@ -13,6 +14,17 @@ module.exports = [
 		},
 	},
 	n.configs["flat/recommended"],
+	{
```

Comment:
[Style] Could you please merge this object with the other `rules`? I.e.:

```ts
	eslint.configs.recommended,
	n.configs["flat/recommended"],
	tsESLint.configs.recommended,
	{
		linterOptions: {
			reportUnusedDisableDirectives: "error",
		},
		rules: {
			"no-undef": "off",
			"no-useless-escape": "off",
			// TS handles this for us
			"n/no-missing-import": "off",
		},
	},
```

I've triaged enough user pain over complicated ESLint configs to have a drive to simplify them as much as possible, whenever possible. 😂

Reply:
Fair enough.  I organized it so that the config turning off the rules was directly after the config that enabled it, but that may not be as obvious and may not even be useful, since rules names have plugin prefixes.